### DISCUSSION TITLE
The apple does not display into the status bar when I have my tmux theme enabled

### DISCUSSION THREAD

**Discussion by billgsm** [olimorris/tmux-pomodoro-plus#21] [Q&A] [answered]
Hello, 

<details><summary>My .tmux.conf</summary>
<p>

```shell
# Use vi key bindings in copy mode
set-option -wg mode-keys vi
set-option -g default-shell /bin/zsh

# first window will have the index 1 instead of 0
set-option -g base-index 1

set-option -g history-limit 10000

# Disable the mouse
set-option -g mouse off

# Reload config and tell it
bind-key r source-file ~/.tmux.conf \; display "Yay reloaded tmux config"

# Pick a window easily with alt-<number>
bind-key -n M-0 select-window -t 0
bind-key -n M-1 select-window -t 1
bind-key -n M-2 select-window -t 2
bind-key -n M-3 select-window -t 3
bind-key -n M-4 select-window -t 4
bind-key -n M-5 select-window -t 5
bind-key -n M-6 select-window -t 6
bind-key -n M-7 select-window -t 7
bind-key -n M-8 select-window -t 8
bind-key -n M-9 select-window -t 9

set-option -g display-time 2000

# Plugins
#########

# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{ 
set -g @plugin 'tmux-plugins/tpm'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}} 

# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{ 
set -g @plugin 'tmux-plugins/tmux-sensible'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}} 

# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{ 
# set -g @plugin 'dracula/tmux'

# ----

set -g @plugin 'arcticicestudio/nord-tmux'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}} 


# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{ 
set -g @plugin 'tmux-plugins/tmux-resurrect'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}}

# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{
set -g @plugin 'sainnhe/tmux-fzf'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}}


# {{{{{{{{{{{{{{{{{{{{{{{{{{{{{
set -g @plugin 'olimorris/tmux-pomodoro-plus'

set-option -ag status-left "#[fg=$color_gray]#{pomodoro_status}"

set-option -g @pomodoro_start 'T' 
set-option -g @pomodoro_cancel 'T'
set-option -g @pomodoro_granularity 'on'
set-option -g @pomodoro_on " 🍅"
set-option -g @pomodoro_complete "#[fg=green]🍅 "
set-option -g @pomodoro_notifications 'on'
set-option -g @pomodoro_sound 'on'
# }}}}}}}}}}}}}}}}}}}}}}}}}}}}}

run '~/.tmux/plugins/tpm/tpm'
``` 

</p>
</details> 

When I start pomodoro, the status bar of tmux does not change 

<details><summary>tmux theme enabled</summary>
<p>

![image](https://github.com/olimorris/tmux-pomodoro-plus/assets/1367298/27a49001-8b62-4889-a112-7c48511fabbf)

</p>
</details>

However, when I remove my theme 'arcticicestudio/nord-tmux' (I also tried dracula/tmux and the result is the same) and I run pomodoro, the apple is shown up

<details><summary>tmux theme disabled</summary>
<p>

![image](https://github.com/olimorris/tmux-pomodoro-plus/assets/1367298/729c459e-11ee-4b58-b0a1-517cf26ca35f)

</p>
</details>

I'm new to tmux too, so I do not know whether I'm missing something somewhere.

Any ideas?

Environment:

```shell
❯ tmux -V
tmux 3.2a  
❯ kitty --version # I've even tried alacritty and ubuntu default terminal and the result is no different
kitty 0.21.2 created by Kovid Goyal
```

**Comment by olimorris** [comment]
Where is `$color_gray` defined?

**Reply by billgsm** [reply]
Nowhere :smirk: , I removed it and now I have this `set-option -ag status-left "#{pomodoro_status}"` but it does not work either.
Same with `set-option -ag status-right "#{pomodoro_status}"`

**Reply by olimorris** [reply]
I can't say I know a great deal about configuring tmux either. My readme never states use append so what happens when you remove that?

This is [how](https://github.com/olimorris/dotfiles/blob/28fece629aadb9b2f39a9eeb940bd0789ce8dc72/.config/tmux/conf/skin_shared.conf#L62) I have it configured in my statusline

**Reply by billgsm** [reply]
> I can't say I know a great deal about configuring tmux either. My readme never states use append so what happens when you remove that?

Nothing changes.

> This is [how](https://github.com/olimorris/dotfiles/blob/28fece629aadb9b2f39a9eeb940bd0789ce8dc72/.config/tmux/conf/skin_shared.conf#L62) I have it configured in my statusline

I'll look into your config files and let you know if it helped. Thanks

**Reply by olimorris** [reply]
Looking at the `nord-tmux` theme, that is one complicated piece of code. Given that the plugin doesn't work with that theme, I'd advise raising an issue/discussion over there too.

**Reply by olimorris** [reply]
Hey @billgsm just thought I'd check in and see if you've had any luck?

**Reply by billgsm** [reply]
Hi, thanks for that. Yeah I opened a gh issue https://github.com/nordtheme/tmux/issues/86 but I haven't got an answer yet.

Anyway, it's more likely something to fix with the theme than with pomodoro as once the theme removed everything works just fine.

Feel free to close this discussion. I'll try to update it if something happens

**Reply by billgsm** [reply]
Hey @olimorris, does this make sense to you https://github.com/nordtheme/tmux/issues/86#issuecomment-1610262068 ?

**Reply by olimorris** [reply, marked as answer]
So Nord Theme requires plugins to integrate with it? Is there a guide for that?

I'd be open to a PR but unlikely to add this myself.
